src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -94,6 +94,7 @@ enum UlduarNPCs
 
     //XT002
     NPC_XS013_SCRAPBOT                      = 33343,
+    NPC_HEART_OF_DESCONSTRUCTOR             = 33329,
```

Comment:
NPC 33329 name is 'Heart of the Deconstructor'
-- you may want to fix the "de**S**constructor" typo

Reply:
Nope
https://reinosdeleyenda.org/aowow/?npc=33329
is a typo xd